include/net/fota_download.h

```diff
@@ -172,14 +171,15 @@ int fota_download(const char *host, const char *file, const int *sec_tag_list,
  *              download, both paths will be treated as upgradable bootloader slot 0
  *              and slot 1 binaries respectively, and only the binary corresponding to
  *              the currently inactive bootloader slot will be selected and downloaded.
-*               See <a href="https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/config_and_build/bootloaders/bootloader.html">
+ *              See <a href="https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/
+ *              config_and_build/bootloaders/bootloader.html">
  *              Secure Bootloader Chain Docs</a> for details regarding the upgradable
  *              bootloader slots.
  * @param sec_tag_list Security tags that you want to use with HTTPS. Pass NULL to disable TLS.
  * @param sec_tag_count Number of TLS security tags in list. Pass 0 to disable TLS.
  * @param pdn_id Packet Data Network ID to use for the download, or 0 to use the default.
  * @param fragment_size Fragment size to be used for the download.
- *			If 0, @kconfig{CONFIG_DOWNLOAD_CLIENT_HTTP_FRAG_SIZE} is used.
+ *			If 0, no fragment is used.
```

Comment:
Align with the rest of the file:

```suggestion
 *			If 0, no fragmentation is used.
```